app/components/settings-modal.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+import {oneWay} from '@ember/object/computed';
+import {inject} from '@ember/service';
+
+export default Component.extend({
+  // Services
+  userSettings: inject(),
+
+  // Properties
+  token_github_com: oneWay('userSettings.tokens.github_com'),
+
+  init() {
+    this._super(...arguments);
+  },
+  actions: {
+    hideModal() {
+      this.set('isActive', false);
+      this.userSettings.setSetting('githubTokenModalSeen', true);
+    },
+    saveSettings() {
+      this.userSettings.setToken('github_com', this.get('token_github_com'));
```

Comment:
This page is expected to have long list of token info when githlab ce is supported. Which is next phase. Save and close no longer be expected.

Reply:
Hmm, I don't understand.

I imagine it would be like this:  
- Show modal with 2 input, GitHub token & GitLab token
- User either fill all of them, or just one, no problem
- User click save, the modal is closed
- Show the tasks from both platform if user filled all tokens, otherwise show from whatever platform that has token.

Reply:
Surely I should add it now. My thought was "When supporting GitHub CE", there will be > 5 inputs, one for each orgs copy and pasting and switch tab is expected user may want to save and then go back to another github ce website to copy token.

I realize that would be a very rare case that people will have account in those 5 repository. I shouldn't think to far for that. Anyway, it was easier to not do that in the code. 😄 .

I will add that.

Reply:
Do you mean GitLab CE? I haven't thought of that :)

But surely it's the same deal, and if there is like >5 inputs, I don't want to hit save after filling `each` input, I just want to hit it once, and then use the app.

Reply:
An annoying thing about the modal is that when it is opened a second time and then closed, the fetched issue list is lost , requiring a refetch.  Is that fixable without much effort?